ui/components/ui/nickname-popover/nickname-popover.component.js

```diff
@@ -27,31 +33,51 @@ const NicknamePopover = ({ address, onClose = null, onAdd = null }) => {
           className="nickname-popover__identicon"
         />
         <div className="nickname-popover__address">
-          {shortenAddress(address)}
+          {nickname || shortenAddress(address)}
         </div>
         <div className="nickname-popover__public-address">
           <div className="nickname-popover__public-address__constant">
             {address}
           </div>
-          <button
-            type="link"
-            onClick={() => {
-              handleCopy(address);
-            }}
+
+          <Tooltip
+            position="bottom"
             title={copied ? t('copiedExclamation') : t('copyToClipboard')}
           >
-            <CopyIcon size={11} color="#989a9b" />
-          </button>
+            <button
+              type="link"
+              onClick={() => {
+                handleCopy(address);
+              }}
+              title=""
+            >
+              <CopyIcon size={11} color="#989a9b" />
+            </button>
+          </Tooltip>
         </div>
+
         <div className="nickname-popover__view-on-block-explorer">
-          {t('viewOnBlockExplorer')}
+          <Button
+            type="link"
+            className="nickname-popover__etherscan-link"
+            onClick={() => {
+              global.platform.openTab({
+                url: explorerLink,
+              });
+            }}
+            target="_blank"
+            rel="noopener noreferrer"
+            title={t('etherscanView')}
```

Comment:
This look un-necessary
target="_blank"
rel="noopener noreferrer"

Reply:
yes I agree, but the same pattern is being used in the codebase for external links in some places, a similar pattern can be found in `/ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js` line 417. happy to remove it if it has no purpose.

Reply:
So these are actually needed.

![Screenshot from 2021-12-01 11-27-32](https://user-images.githubusercontent.com/7499938/144257598-166ddb57-de87-45ed-b554-b6d3d1876942.png)

Reply:
Good point @jpuri
I created a ticket for this: https://github.com/MetaMask/metamask-extension/issues/12923